packages/client/src/constants.js

```diff
@@ -7,14 +9,139 @@ export const COLORS = {
   lightgray: '#EDEDED',
   lightorange: '#FFF6DE',
   lightblue: '#E7F2FB',
+  black: '#000000',
 };
 
 export const FONTS = {
   font1: "'Raleway', sans-serif",
   font2: "'Montserrat', sans-serif",
-  black: '#000000',
 };
 
+export const FAQS = [
+  {
+    key: "0",
+    category: "mentee",
+    question: "Who is eligible to be mentored?",
+    answer: <div>All K-12 students in the US who are currently struggling with classes are eligible to be matched with an undergraduate or postgraduate mentor for personalized mentorship and academic support. We are especially committed to supporting underprivileged students. For students under the age of 18, we require that a parent or guardian be present during all mentoring sessions.</div>
+  },
+  {
+    key: "1",
+    category: "general",
+    question: "What services are available?",
+    answer: <div>Our mentors provide tutoring services for K-12 school subjects, including many Honors and AP classes. Additionally, some of our mentors are willing to help with college preparation, such as standardized testing (SAT, ACT, AP, etc.).</div>
+  },
+  {
+    key: "2",
+    category: "general",
+    question: "How does mentoring work?",
+    answer: <div>Mentoring sessions will be conducted online via video chat. We suggest using Skype, Google Hangouts, or Zoom, although the the decision is up to the parent or guardian, student, and mentor.</div>
+  },
+  {
+    key: "3",
+    category: "mentee",
+    question: "What if I do not have access to Wi-Fi?",
+    answer: <div>Spectrum is providing free Wi-Fi services for students during the school shutdown. Their number is 1-844-488-8398. Mentoring lessons can also be held via phone call.</div>
+  },
+  {
+    key: "4",
+    category: "mentee",
+    question: "How do I get matched with a mentor? How does the matching process work?",
+    answer: <div>After filling out the registration form and verifying their emails, parents/guardians will be able to access our 'Find a Mentor' page to find a mentor for their child. From there, they will be able to send messages to any of the mentors in our database. Please do not request more than 1 mentor per student per 24 hour period.</div>
+  },
+  {
+    key: "5",
+    category: "mentee",
+    question: "How many mentors can each student have?",
+    answer: <div>Each student is paired with one mentor.</div>
+  },
+  {
+    key: "6",
+    category: "mentee",
+    question: "What if I have multiple students I want to request mentors for?",
+    answer: <div>Parents/guardians are able to request up to 4 students, with a maximum of one mentor request per student. Be sure to mention the name of the student you're requesting for.</div>
```

Comment:
I don't think 'request up to 4 students' is the right phrase, they're requesting mentors not students. Maybe 'Parents can request up mentors for up to 4 students (one per student).'